.env.example

```diff
@@ -1 +1,4 @@
-MY_SECRET="MY_SECRET"
+X25519_PRIVATE_KEY=TNjblUQ1U959TKGddmNlyOJQ7RMGaqS0XXH185Ei1Ik
```

Comment:
Usually you leave values empty in the example.

Reply:
I was trying to make it easy for testing/debugging and it's the anvil account so security isn't a concern but I'll update this